pkg/spanconfig/spanconfigsqlwatcher/buffer.go

```diff
@@ -119,27 +185,94 @@ func (b *buffer) flushEvents(
 // combined frontier timestamp is also returned.
 func (b *buffer) flush(
 	ctx context.Context,
-) (updates []spanconfig.DescriptorUpdate, _ hlc.Timestamp, _ error) {
+) (sqlUpdates []spanconfig.SQLUpdate, _ hlc.Timestamp, _ error) {
 	events, combinedFrontierTS := b.flushEvents(ctx)
-	sort.Slice(events, func(i, j int) bool {
-		ei, ej := events[i].(event), events[j].(event)
-		if ei.update.ID == ej.update.ID {
-			return ei.timestamp.Less(ej.timestamp)
-		}
-		return ei.update.ID < ej.update.ID
+	bufferedEvents := make(bufferedEvents, 0, len(events))
```

Comment:
[nit] s/events/bufferedEvents and s/bufferedEvents/events -- the latter is more precious.
[nit] s/type bufferedEvents/type events

Reply:
did s/events/bufferedEvents and s/type bufferedEvents/type events

but the actual slice we care about cant shadow the type name according to lint 🤷 so called it something other than `events`.